src/utils/load-audio.tsx

```diff
@@ -0,0 +1,34 @@
+/** @jsx createElementEntity */
+import { createElementEntity } from "../utils/jsx-entity";
+import { ProjectionMode } from "./projection-mode";
+import { VideoTexture } from "three";
+import { renderAsEntity } from "../utils/jsx-entity";
+import { loadAudioTexture } from "../utils/load-audio-texture";
+import { HubsWorld } from "../app";
+
+export function* loadAudio(world: HubsWorld, url: string) {
+  const { texture, ratio }: { texture: VideoTexture; ratio: number } = yield loadAudioTexture(url);
+
+  // TODO: VideoTexture.image must be content that be played
+  //       in video-system. And it is also used to render.
+  //       It is audio here so the object will be rendered as
+  //       black. Audio icon must be rendered. Fix this.
```

Comment:
If it helps, we have icons in the video menu system that can be used as an example for how to do this. You could add one on each side of the entity below (as children) to display an image.

Reply:
a single double sided material would be better. In the old code we use this https://github.com/mozilla/hubs/blob/aaa4584a822545a8ea998c3c14edc62078fb542b/src/components/media-video.js#L30

Ideally we would just do that in an inflator for "audio" and change how we are getting the video element reference so we don't need to do any weird workaround.